packages/babel-plugin-minify-mangle-names/README.md

```diff
@@ -71,3 +71,4 @@ require("babel-core").transform("code", {
 + `blacklist` - A plain JS Object with keys as identifier names and values indicating whether to exclude
 + `eval` - mangle identifiers in scopes accessible by eval
 + `keepFnName` - prevent mangler from altering function names. Useful for code depending on `fn.name`
++ `topLevel` - mangle topLevel Identifiers
```

Comment:
To me, it feels like this name should be something we generalize across plugins for standalone scripts, vs commonjs scripts. Thoughts? `topLevel` applies to the case of the mangler, but for example in the case we heard about on slack, it was the top-level behavior of DCE that was failing. `topLevel` could apply to both, but the name itself doesn't really say much about _why_ top-level bindings should/should not be treated as special cases.

How would you feel about this being `standaloneScript` with a default of `false`? So if people want to use Babili on a whole script file, rather than on individual CommonJS/ES6 modules, they'd have to opt into it?